src/main/java/com/altoros/aion/Ownable.java

```diff
@@ -4,14 +4,9 @@
 import avm.Blockchain;
 import org.aion.avm.tooling.abi.Callable;
 
-import static com.altoros.aion.Ownable.Const.NO_ADDRESS;
-import static com.altoros.aion.Ownable.Const.OWNERSHIP_TRANSFERRED;
-
 public class Ownable {
-    public static class Const {
-        public static final Address NO_ADDRESS = new Address(new byte[32]);
-        public static final String OWNERSHIP_TRANSFERRED = "OwnershipTransferred";
-    }
+    public static final Address NO_ADDRESS = new Address(new byte[32]);
+    public static final String OWNERSHIP_TRANSFERRED = "OwnershipTransferred";
```

Comment:
As part of the review, it was suggested to use `byte[]` instead of `string` because it is cheaper. Is it possible to apply this changes and adapt unit tests to properly use it if needed?
```
 private static final byte[] OWNERSHIP_TRANSFERRED = "OwnershipTransferred".getBytes();
```

Reply:
As this change require addition test refactoring it should be done separately. This PR was quick fix.